src/chipeitlib/src/main/java/es/chipeit/lib/emulator/Chipeit.kt

```diff
@@ -23,19 +20,19 @@ class Chipeit(
     val SCREEN_WIDTH = 64
     val SCREEN_HEIGHT = 32
 
-    private val _graphicsMemory = ByteArray(SCREEN_WIDTH * SCREEN_HEIGHT)
-    val PUBLIC_GRAPHICS_MEMORY = ByteArray(SCREEN_WIDTH * SCREEN_HEIGHT)
+    private val _graphicMemory = GraphicMemory(Array(SCREEN_HEIGHT) { Array(SCREEN_WIDTH) { false } })
+    val graphicMemory: IGraphicMemory = _graphicMemory
 
     @Volatile
     private var running: Boolean = false
     private val memory = LoggedMemory(
             "Main memory",
-            PaddedMemory(ByteMemory(romContent), 0x200)
-    )
-
-    private val graphicMemory = LoggedMemory(
-            "Graphic memory",
-            ByteMemory(_graphicsMemory)
+            ByteMemory(
+                    TABLE +
+                            ByteArray(0x200 - TABLE.size) +
```

Comment:
Change the `0x200` to a constant somewhere, and use it for Cpu initialization too.